src/json/next/serialization.cr

```diff
@@ -2,6 +2,109 @@ module JSON
   annotation Field
   end
 
+  # The `JSON::Serializable` module is a better alternative to `JSON.mapping`.
+  #
+  # ### Example
+  #
+  # ```
+  # require "json"
+  #
+  # class Location
+  #   include JSON::Serializable
+  #   property lat : Float64
```

Comment:
this is probably a perfect place to show off `@[JSON::Field(key: "lat")]` and rename this property `latitiude`. Same for `longitude`.